examples/reminders/lib/feature_reminder_list/ui_components/scroll_to_index_widget.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_sticky_header/flutter_sticky_header.dart';
+import 'package:rx_bloc_list/models.dart';
+import 'package:scroll_to_index/scroll_to_index.dart';
+
+import '../../app_extensions.dart';
+import '../../base/common_ui_components/app_reminder_tile.dart';
+import '../../base/common_ui_components/app_sticky_header.dart';
+import '../../base/models/reminder/reminder_model.dart';
+
+class ScrollToPositionWidget extends StatefulWidget {
+  const ScrollToPositionWidget({
+    required this.remindersList,
+    this.createdReminderId,
+    Key? key,
+  }) : super(key: key);
+
+  final PaginatedList<ReminderModel> remindersList;
+  final String? createdReminderId;
```

Comment:
It will be great to express the intent instead of the concrete use case. From what I see if we pass the createdReminderId, the widget will scroll to the matching reminder, so you can rename this prop from `createdReminderId` to `scrollToReminderId`

Reply:
The name was changed from `createdReminderId `to `scrollToReminderId`